packages/block-editor/src/store/reducer.js

```diff
@@ -1650,6 +1651,30 @@ export function highlightedBlock( state, action ) {
 	return state;
 }
 
+export function documentSettingsDropdown(
+	state = { hoveredEntities: [] },
+	action
+) {
```

Comment:
This extra level may not be necessary, but I was thinking it may be helpful if we need other items tracked in central state than just the hovered items.

Also, this may not be the correct store.. perhaps a different store or a new one specifically for the document settings dropdown may make more sense?

Reply:
Yeah I could definitely see needing a store for document settings at some point. I also agree that it probably isn't the right store. Depending on how things go, I think a nested reducer in the editor stores might be the most obvious place :thinking: